src/webots/gui/WbShareWindow.cpp

```diff
@@ -33,41 +34,38 @@ WbShareWindow::WbShareWindow(QWidget *parent) : QDialog(parent) {
     uploadUrl = uploadUrl.split("//")[1];
   groupBoxStyleSheet = "QGroupBox {border: 1px solid gray;border-radius: 9px;margin-top: 0.5em; } QGroupBox::title "
                        "{subcontrol-origin:  margin; subcontrol-position: top center; }";
-  this->setWindowTitle(tr("Share your simulation online"));
+  this->setWindowTitle(tr("Share your simulation"));
 
   QGridLayout *layout = new QGridLayout(this);
 
-  QLabel *labelIntro = new QLabel(this);
-  labelIntro->setOpenExternalLinks(true);
-  labelIntro->setText(tr("<html><head/><body><p>Publish your simulation on <a href=\"https://%1/\"><span "
-                         "style=\" text-decoration: underline; color:#5dade2;\">%1</span></a>.</p></body></html>")
-                        .arg(uploadUrl));
-  layout->addWidget(labelIntro, 3, 0, 1, 2);
+  QRadioButton *boxButtons[2];
+  boxButtons[0] = new QRadioButton("Upload to " + uploadUrl, this);
```

Comment:
The only thing that is worse than the previous version is that now the webots.cloud is a simple text and no longer a rich text with link to URL.
But it is seems complicated to enable rich HTML text in radio label, so it's fine.